.travis.yml

```diff
@@ -4,7 +4,6 @@ python:
   - 3.3
   - 3.4
   - 3.5
-  - 3.6
```

Comment:
Travis CI has pre-releases of the next upcoming Python version called "nightly": https://docs.travis-ci.com/user/languages/python#Choosing-Python-versions-to-test-against

That doc also implies that "3.6-dev" *might* exist.